src/webextension/l10n.js

```diff
@@ -9,16 +9,16 @@ import { LocalizationProvider } from "fluent-react";
 import PropTypes from "prop-types";
 import React, { Component } from "react";
 
-async function fetchMessages(baseDir, locale) {
-  const response = await fetch(`${baseDir}/${locale}.ftl`);
+async function fetchMessages(baseDir, locale, bundle) {
+  const response = await fetch(`${baseDir}/${locale}/${bundle}.ftl`);
   const messages = await response.text();
 
   return { [locale]: messages };
 }
 
-async function createMessagesGenerator(baseDir, currentLocales) {
+async function createMessagesGenerator(baseDir, currentLocales, stringBundle) {
   const fetched = await Promise.all(
-    currentLocales.map((x) => fetchMessages(baseDir, x))
+    currentLocales.map((x) => fetchMessages(baseDir, x, stringBundle))
   );
   const bundle = fetched.reduce(
```

Comment:
Nit: let's call this variable `bundles` now, since it's technically a collection of bundles (one for each of `currentLocales`).

Reply:
Good idea.

That also lets me change `stringBundle` to `bundle`, for consistency and clarity. 👍